### PR TITLE
FileClayblock

### LINKED ISSUE #11: FileClayblock

### PR DESCRIPTION
**Pull request by Jagni**
Fixes #11 

Ideia de melhoria: Mostrar uma label ao invés do link.
Bloqueio: DataClayblock só permite inserir type e value.
Sugestão: poderíamos adicionar um linkedHashmap pra inserir argumentos extras maybe?

### CODE REVIEW

File: examples/development_app/ios/Podfile [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+def parse_KV_file(file, separator='=')
+  file_abs_path = File.expand_path(file)
+  if !File.exists? file_abs_path
+    return [];
+  end
+  pods_ary = []
+  skip_line_start_symbols = ["#", "/"]
+  File.foreach(file_abs_path) { |line|
+      next if skip_line_start_symbols.any? { |symbol| line =~ /^\s*#{symbol}/ }
+      plugin = line.split(pattern=separator)
```
Comment (RegisMelgaco) [inline examples/development_app/ios/Podfile]: Codacy Issue found: [Useless assignment to variable - `pattern`.](https://app.codacy.com/app/Red-Ovenbirds/taiper/pullRequest?prid=4544453)

**Comment by RegisMelgaco** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- examples/development_app/ios/Podfile  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/taiper/taiper/pullRequest?prid=4544453&bid=15291402)

File: examples/development_app/ios/Podfile [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+project 'Runner', {
+  'Debug' => :debug,
+  'Profile' => :release,
+  'Release' => :release,
+}
+
+def parse_KV_file(file, separator='=')
+  file_abs_path = File.expand_path(file)
+  if !File.exists? file_abs_path
```
Comment (RegisMelgaco) [inline examples/development_app/ios/Podfile]: Codacy Issue found: [Favor `unless` over `if` for negative conditions.](https://app.codacy.com/app/Red-Ovenbirds/taiper/pullRequest?prid=4544453)

File: examples/development_app/ios/Podfile (hunk 1 above)
Comment (Jagni) [reply examples/development_app/ios/Podfile]: É código do podfile gerado e mantido pelo Flutter... Num pode mexer

**Comment by RegisMelgaco** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- examples/development_app/ios/Podfile  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/taiper/taiper/pullRequest?prid=4544453&bid=15291402)

File: examples/development_app/ios/Podfile (hunk 1 above)
Comment (RegisMelgaco) [inline examples/development_app/ios/Podfile]: Codacy Issue found: [Useless assignment to variable - `pattern`.](https://app.codacy.com/app/Red-Ovenbirds/taiper/pullRequest?prid=4544453)

File: examples/development_app/ios/Podfile (hunk 2 above)
Comment (RegisMelgaco) [inline examples/development_app/ios/Podfile]: Codacy Issue found: [Favor `unless` over `if` for negative conditions.](https://app.codacy.com/app/Red-Ovenbirds/taiper/pullRequest?prid=4544453)

**Comment by RegisMelgaco** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- examples/development_app/ios/Podfile  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/taiper/taiper/pullRequest?prid=4544453&bid=15291402)

**Review by RegisMelgaco — APPROVED** [review]

**Comment by RegisMelgaco** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- examples/development_app/ios/Podfile  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/taiper/taiper/pullRequest?prid=4544453&bid=15291402)

File: examples/development_app/ios/Podfile (hunk 1 above)
Comment (RegisMelgaco) [inline examples/development_app/ios/Podfile:22]: Codacy Issue found: [Useless assignment to variable - `pattern`.](https://app.codacy.com/app/Red-Ovenbirds/taiper/pullRequest?prid=4544453)

File: examples/development_app/ios/Podfile (hunk 2 above)
Comment (RegisMelgaco) [inline examples/development_app/ios/Podfile:15]: Codacy Issue found: [Favor `unless` over `if` for negative conditions.](https://app.codacy.com/app/Red-Ovenbirds/taiper/pullRequest?prid=4544453)

**Pull request merged**